integration-tests/tests/pages/review.page.ts

```diff
@@ -17,4 +27,43 @@ export class ReviewPage {
         await this.page.getByRole('button', { name: /Release \d+ valid sequence/ }).click();
         await this.page.getByRole('button', { name: 'Release', exact: true }).click();
     }
+
+    async viewSequences() {
+        const button = this.viewSequencesButton();
+        await expect(button).toBeVisible({ timeout: 30000 }); // Increase timeout to allow for processing
```

Comment:
I think it'd be better if we had an explicit step in the test "wait for sequence processing to be finished" - this might well test for the presence of the button - but I think having this timeout here is unexpected.

I think we should keep timeouts to a minimum, and the ones we do have should be made quite explicit in the tests.

Reply:
I agree this could be good but would prefer not to tackle it in this PR